WcaOnRails/db/migrate/20180831164420_add_z1_and_z3_columns_to_competitions.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddZ1AndZ3ColumnsToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :regulation_z1, :boolean, null: true, default: nil
+    add_column :Competitions, :regulation_z1_reason, :string, null: true, default: nil
+
+    add_column :Competitions, :regulation_z3, :boolean, null: true, default: nil
+    add_column :Competitions, :regulation_z3_reason, :string, null: true, default: nil
```

Comment:
Shouldn't this be `text` (or perhaps `mediumtext`) like the `competitor_limit_reason` column? I don't know how long these are expected to be, just want to ensure `string` is intended =)

Reply:
Yes, it will be better to have it like text.